code/__DEFINES/mobs.dm

```diff
@@ -0,0 +1,3 @@
+//Grasp indexes
```

Comment:
I don't have a problem with this from an organizational standpoint, but if this is going to be a thing you should move all the mob specific defines from setup.dm to this file.